pkg/monitortests/kubeapiserver/auditloganalyzer/monitortest.go

```diff
@@ -390,7 +390,7 @@ func (w *auditLogAnalyzer) EvaluateTestsFromConstructedIntervals(ctx context.Con
 		}
 	}
 
-	testName = "API LBs follow /readyz of kube-apiserver and stop sending requests before server shutdowns for external clients"
+	testName = "[sig-api-machinery][Feature:APIServer][Late] API LBs follow /readyz of kube-apiserver and stop sending requests before server shutdowns for external clients [Suite:openshift/conformance/parallel]"
```

Comment:
Thanks for deduping. Small problems, I believe the late and suite tags in the test name are now quite misleading and inaccurate. And yet if a rename is done it needs to be reflected in the mapping repo. 

Lets make it [sig-api-machinery][Feature:APIServer] API LBs follow /readyz of kube-apiserver and stop sending requests before server shutdowns for external clients

I will handle the test rename mapping in this instance over in https://github.com/openshift-eng/ci-test-mapping

Reply:
Updated